department-of-veterans-affairs/caseflow
# Kev ma/appeals 55015

Resolves [https://jira.devops.va.gov/browse/APPEALS-55015](https://jira.devops.va.gov/browse/APPEALS-55015)

# Description
Updated how organizationPermissions and organizationUserPermissions are loaded into the frontend from server side props in ruby views to React component fetch on page mount for OrganizationUsers.jsx

## Acceptance Criteria

## Testing Plan
1. Sign in as a Team Admin user
2. Click on team management in the User dropdown
3. Click the Org Admin button for InboundOpsTeam
4. Note that the error page no longer appears and you are successfully directed to organization users page
5. Inspect the page through Chrome Dev Tools and React Dev Tools, and note that the data for permissions is passed to OrganizationUsers.jsx and OrganizationPermissions.jsx

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/6a60d5db-53b3-4280-8819-7b932b3fd513">

|


https://github.com/user-attachments/assets/0975b248-e2e0-4956-ae5f-8167b5d513d7



# Backend
# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
